app/views/admin/graphs/index.html.haml

```diff
@@ -1,70 +1,60 @@
 
 / inline scss is wrong and shouldn't be done - but because we're avoiding adding new things to the existing admin stylesheets, putting it here anyway
 / TODO: put into admin stylesheets when we update admin stylesheets
```

Comment:
I think you can delete these comments, since they've been resolved? There is now scss in the new scss 🙂